src/lib/components/rentComponents/rentTable.tsx

```diff
@@ -0,0 +1,96 @@
+import IRent from "../../entities/IRent";
+
+interface RentTableRowProps {
+  rents: IRent[];
+  onModify: (rent: IRent) => void;
+  onDelete: (rentId: number) => void;
+}
+
+const RentTableRow: React.FC<RentTableRowProps> = ({
+  rents,
+  onModify,
+  onDelete,
+}) => {
+  const handleModify = (rent: IRent) => {
+    onModify(rent);
+  };
+  const handleRedirect = (path: string) => {
+    window.location.href = `http://localhost:3000${path}`;
```

Comment:
Let's use this url from an env var, so it can be easy to update if your app is running in a different environment, You can do it by implementing [dotenv](https://www.npmjs.com/package/dotenv)